ui/src/assets/widgets/button.scss

```diff
@@ -14,7 +14,7 @@
 
 @import "theme";
 
-.pf-button {
+button.pf-button {
```

Comment:
Why is the prefix `button` being added here? In the subsequent file, you didn't change `.pf-popup` to `popup.pf-popup`, so there must be something special going on here. It's not obvious to me.

Reply:
There is a CSS selector that within Perfetto proper only selects html buttons, but when the "perfetto" class selector is added to it by the build script then the "pf-button" class selector is a lower priority then it should be, so I needed to make the pf-button more specific so that it would take priority.

Reply:
A better solution might be to be more selective about which CSS selectors are skipped by default, for instance just one's that involve the pop-up functionality.  Or I could even add the Perfetto class to the pop-ups at their top level.

Reply:
The latter sounds like a better solution to me unless you think a number of different UI elements are suffering the same problem.